src/schema/v2/artwork/index.ts

```diff
@@ -809,6 +809,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           )
             return `Shipping: ${domesticShipping} ${shippingRegion} only`
 
+          if (
+            artwork.domestic_shipping_fee_cents == null &&
+            artwork.international_shipping_fee_cents !== null
+          )
```

Comment:
Hi:) Happy to discuss it in person but I think this is not a true statement here. Don't think we would ever apply international shipping cents to domestic shipping so the phrase `worldwide` is misleading here. 

This state should never be possible for bnmo works (ui constraints supposed to prevent it in CMS) and for non bnmo works we were never using/displaying this data. As we are starting to do it now for inquiry works - I think we need to reconsider validations on artwork and decide what exactly we want to show on inquiry works.

As far as this pr concerned - I think if we don't want to display `Shipping calculated at checkout here` the more proper solution here is to return empty string in case domestic is not present but international is present.

Reply:
oooh, got it, many thanks for all the context, and sorry about the confusion!! 🙇‍♀️ 
I think `Shipping calculated at checkout` makes sense at this point, so we can keep consistency with the other scenarios. Just made this change and squashed it.

Reply:
This change is better but I'm wondering if the more proper way to handle it would be updating the first couple of line of this method and using this newly added `process_with_arta_shipping`.

So something like:
```
resolve: (artwork) => {
          if (artwork.process_with_arta_shipping) {
            return "Shipping: Calculated in checkout"
          }

          if (artwork.domestic_shipping_fee_cents == null)
            return "Shipping, tax, and additional fees quoted by seller"
      
          //and the test is without change from master....
```

I *think* it would most accurately describe what will happen to shipping. If the work is arta or we'll silently opt it into Arta - we can say that collector can see shipping at checkout.
The artwork is not complete - i.e. missing domestic shipping - we'll let partner define what is the correct shipping will be and the incomplete offer will be generated. 

Sorry for all the back and force @lidimayra . I'm just trying to figure out what is actually true for collector here.

Reply:
No apologies needed, I'm actually thankful for your patience, I'm learning tons in the process!! ❤️ 

Just to confirm I got it: we were already displaying`Shipping, tax, and additional fees quoted by seller` message in case `artwork.domestic_shipping_fee_cents` is null and `artwork.international_shipping_fee_cents` is also null. Apparently, we don't actually care about `artwork.international_shipping_fee` in order to control this message, right? In case `artwork.international_shipping_fee` is null, this condition must be enough to display this, so I changed the existing example to remove the unwanted part. And this won't ever be reached in case artwork will be processed with Arta, because in this scenario, it won't be quoted by seller, it will be quoted according to Arta shipping option.

Please, let me know in case this doesn't sound good.